packages/js/src/indexables-page/indexables-page.js

```diff
@@ -775,7 +790,13 @@ function IndexablesPage( { setupInfo } ) {
 										addToIgnoreList={ setIgnoredIndexable }
 										position={ position }
 									>
-										<IndexableScore colorClass={ mostLinkedAssessment( indexable ) } />
+										<div className="yst-flex yst-items-center">
+											{
+												( !! parseInt( indexable.is_cornerstone, 10 ) === true )
```

Comment:
For readability this is pretty ok, though you could also go with `=== "1"` right?

Reply:
Yep